web/server/vue-cli/src/components/Report/ReportFilter/ReportCount.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <span class="pa-1 text-no-wrap">
+  <span class="pa-1 text-no-wrap" title="Outstanding reports">
```

Comment:
This counter shows not only the outstanding reports, but the number of all reports that match the filter. "Number of reports" would be more adequate if any.